### DISCUSSION TITLE
Stuck in startmenu when opening skin editor while beatmap with no objects is selected

### DISCUSSION THREAD

**Discussion by tanoshi40** [ppy/osu#26363] [Q&A] [answered]
Steps to reproduce
- select a map with no objects (eg. some map you just created in the editor)
- open skin editor

after that I get an error and come back to the menu, however i cannot click anything (except settings)
the only way after that to quit or restart the game is via the taskmanager or taskbar from what I see

This is for osu!lazer tho, not sure if this is the right repo

https://github.com/ppy/osu-stable-issues/assets/132229391/f3486a10-b350-425a-a36f-6f0004aa2f00

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/25877
